Content.Server/_DV/CosmicCult/CosmicGlyphSystem.cs

```diff
@@ -0,0 +1,132 @@
+using Content.Server.Popups;
+using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CosmicCult.Components.Examine;
+using Content.Shared.Damage;
+using Content.Shared.Examine;
+using Content.Shared.Humanoid;
+using Content.Shared.Interaction;
+using Content.Shared.Mobs.Systems;
+using Robust.Server.Audio;
+using Robust.Shared.Audio;
+using Robust.Shared.Containers;
+
+namespace Content.Server._DV.CosmicCult;
+
+public sealed class CosmicGlyphSystem : EntitySystem
+{
+    [Dependency] private readonly DamageableSystem _damageable = default!;
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly PopupSystem _popup = default!;
+    [Dependency] private readonly MobStateSystem _mobState = default!;
+    [Dependency] private readonly AudioSystem _audio = default!;
+    [Dependency] private readonly SharedContainerSystem _container = default!;
+
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<CosmicGlyphComponent, ExaminedEvent>(OnExamine);
+        SubscribeLocalEvent<CosmicGlyphComponent, ActivateInWorldEvent>(OnUseGlyph);
+    }
+
+    #region Base trigger
+
+    private void OnExamine(Entity<CosmicGlyphComponent> uid, ref ExaminedEvent args)
+    {
+        if (HasComp<CosmicCultComponent>(args.Examiner))
+        {
+            args.PushMarkup(Loc.GetString("cosmic-examine-glyph-cultcount", ("COUNT", uid.Comp.RequiredCultists)));
+            switch (uid.Comp.GlyphName) // This seems like the most straightforward way to do this, rather than making seperate examine events for each glyph component
+            {
+                case "knowledge":
+                    args.PushMarkup(Loc.GetString("cosmic-examine-glyph-knowledge"));
+                    break;
+                case "truth":
+                    args.PushMarkup(Loc.GetString("cosmic-examine-glyph-truth"));
+                    break;
+                case "cessation":
+                    args.PushMarkup(Loc.GetString("cosmic-examine-glyph-cessation"));
+                    break;
+                case "blades":
+                    args.PushMarkup(Loc.GetString("cosmic-examine-glyph-blades"));
+                    break;
+                case "warding":
+                    args.PushMarkup(Loc.GetString("cosmic-examine-glyph-warding"));
+                    break;
+                case "projection":
+                    args.PushMarkup(Loc.GetString("cosmic-examine-glyph-projection"));
+                    break;
+            }
+        }
+        else
+            args.PushMarkup(Loc.GetString("cosmic-examine-text-glyphs"));
+    }
```

Comment:
Add braces for clarity

```suggestion
        else
        {
            args.PushMarkup(Loc.GetString("cosmic-examine-text-glyphs"));
        }
    }
```